### PR TITLE
Create Discover hosts doing possible network scans

### PR DESCRIPTION
**Pull request by anthonws**

### CODE REVIEW

**Comment by meni-braun** [comment]
Please change the file extension to `.txt`, as other queries.

File: Discovery/Discover hosts doing possible network scans [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//Looking for high volume queries against a given RemoteIP, per ComputerName, RemotePort and Process
+//Please change the EventTime window according your preference/objective, as also the subnet ranges that you want to analyze against
+NetworkCommunicationEvents
+| where EventTime > ago(1d) and RemoteIP startswith "172.16" or RemoteIP startswith "192.168" 
+| summarize
+    by ComputerName, RemoteIP, RemotePort, InitiatingProcessFileName
+| summarize RemotePortCount=dcount(RemotePort) by ComputerName, RemoteIP, InitiatingProcessFileName
+//Please change the min value, for a host reaching out to remote ports on a remote IP, that you consider to be threshold for a suspicious behavior 
+| where RemotePortCount > 10
```
Comment (meni-braun) [inline Discovery/Discover hosts doing possible network scans]: Consider to define this value as a `let remotePortCountThreshold = 10;` at the beginning of the query, and then use it here.

File: Discovery/Discover hosts doing possible network scans [hunk 2]
```diff
@@ -0,0 +1,9 @@
+//Looking for high volume queries against a given RemoteIP, per ComputerName, RemotePort and Process
```
Comment (meni-braun) [inline Discovery/Discover hosts doing possible network scans]: Our convention is to use `// ` followed by a space.

**Comment by anthonws** [comment]
Hi @meni-braun ! Looks like you already changed the extension (.txt) and also added a let for the distinct port count threshold (Thanks a lot!)!

Do I need to do anything else?

**Comment by meni-braun** [comment]
Nope, your contribution is merged and available.
Thank you @anthonws!

**Pull request merged**